platform/qt/qt.cmake

```diff
@@ -1,9 +1,18 @@
 # Note: Using Sqlite instead of QSqlDatabase for better compatibility.
 
+option(MBGL_WITH_QT_DEMO "Build Mapbox GL Qt demo" ON)
+option(MBGL_WITH_QT_TEST "Build Mapbox GL Qt tests" ON)
+option(MBGL_WITH_QT_HEADLESS "Build Mapbox GL Qt with headless support" ON)
```

Comment:
Isn't there a way to call the install rule for just a single target?

Reply:
Problem is that even if I just want to build qmapboxgl, I get dependencies pulled into it that I don't want to have. For example, Qt::Widgets are not used on our embedded platform (Sailfish OS). So, it was the best way I found to make it possible.